docs/launch_boot_tests.py

```diff
@@ -112,6 +112,9 @@
                         run = gem5Run.createFSRun(
                             'gem5/build/X86/gem5.opt',
                             'configs-boot-tests/run_exit.py',
+                            '''results/X86/run_exit/vmlinux-5.2.3/
+                            boot-exit/{}/{}/{}/{}
+                            '''.format(cpu, mem, num_cpu, boot_type),
```

Comment:
Do you mean to be hardcoding linux version 5.2.3? If so, why do you create multiple kernel artifacts above?

Also, when creating the kernel artifacts above, you're actually creating 4 that are exactly the same.

Reply:
No, I meant to iterate through all kernels. Fixed in a new commit